Components/Grid/grid.css

```diff
@@ -0,0 +1,17 @@
+@import url("../css/global.css");
+
+.display-grid{
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(10rem,1fr));
+    gap: 1rem;
+}
+
+.box{
+    background-color: aqua;
```

Comment:
Try using utility classes instead of using a repeated color name . e.g tomato color is used at multiple places